### ISSUE TITLE
problem about RELU inplace

### ISSUE THREAD

**Issue by huangjch526** [mli0603/stereo-transformer#61]
```
def _relu_inplace(self):
        """
        make all ReLU inplace
        """
        for m in self.modules():
            if isinstance(m, nn.BatchNorm2d):
                m.inplace = True

```
In your code module/sttr.py, I find a problem in function _relu_inplace(), why the nn.BatchNorm2d is set instead of RELU?

**Comment by mli0603** [comment]
fixed in commit (https://github.com/mli0603/stereo-transformer/commit/44703b01d59c8accff1d396ec842c3d1b9a8fb9a)
